server.go

```diff
@@ -3955,6 +3953,18 @@ func (s *Server) runContinuousQuery(cq *ContinuousQuery) {
 	cq.mu.Lock()
 	defer cq.mu.Unlock()
 
+	// Get server CQ parameters under lock, to avoid races.
+	s.mu.RLock()
```

Comment:
do we really need to lock on these? They can't ever get updated on a running process. They're set when the server object is created and never set again.